### DISCUSSION TITLE
Screen protection over "recent apps" and "screenshot"

### DISCUSSION THREAD

**Discussion by serrq** [mullvad/mullvadvpn-app#8936] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

Some apps give the ability (optional) to switch ON/OFF the "screenshot" and "recent apps" protection feature by other prying apps.
Basically a privacy feature. 

Here an implementation and some examples.

![1000108037](https://github.com/seemoo-lab/AirGuard/assets/113377431/15129f78-1ae2-4431-bb72-5ef60a454d6e)

![1000042036](https://github.com/horizontalsystems/unstoppable-wallet-android/assets/113377431/499a76af-6db7-4f6b-b886-f06ff7dad708)

### Alternative solutions

Privacy enhancement.

All platform involved.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [X] Android
- [X] iOS
- [X] Windows
- [X] macOS
- [X] Linux

**Comment by Pururun** [comment]
Thank you for your suggestion, I have created an internal issue for it.
